datafusion/core/src/physical_optimizer/dist_enforcement.rs

```diff
@@ -2104,7 +2080,6 @@ mod tests {
         // The optimizer should not add an additional SortExec as the
         // data is already sorted
         let expected = &[
```

Comment:
Why was the `SortPreservingMergeExec` removed? Is it because there is a single partition in the input? 

I think the intent of this test was to ensure when a `SortPreservingMergeExec` a new sort was not added.

Would it be possible to update the test so that the `SortPreservingMergeExec` is still present (perhaps by increasing the number of input partitions)? 

Perhaps using the same strategy of `parquet_exec_multiple_sorted`

Reply:
Thanks for pointing this out. I have converted input source to `parquet_exec_multiple_sorted` test now keeps `SortPreservingMergeExec`.